File: python/7 kyu/254.py
# https://www.codewars.com/kata/529eef7a9194e0cbc1000255/train/python

# Passed

from collections import Counter

def is_anagram(test, original):
    test_letter_counter = Counter(test.lower())
    original_letter_counter = Counter(original.lower())
    result = test_letter_counter == original_letter_counter
    return result

output = is_anagram("foefet", "toffee")
print(output)

File: nodejs/7 kyu/254.js
// https://www.codewars.com/kata/559590633066759614000063/train/javascript

// Passed

function minMax(arr) {
  let buyPrice = Math.min(...arr);
  let sellPrice = Math.max(...arr);
  let result = [buyPrice, sellPrice];
  return result;
}

const output = minMax([1, 2, 3, 4, 5]);
console.log(output);